docs/examples/tutorial/cython_tutorial/primes.py

```diff
@@ -0,0 +1,27 @@
+def primes(nb_primes: cython.int):
+    i: cython.int
+    p: cython.int[1000]
+
+    if nb_primes > 1000:
+        nb_primes = 1000
+
+    if not cython.compiled:
+        p = [0] * 1000
+
+    len_p: cython.int = 0  # The current number of elements in p.
+    n: cython.int = 2
+    while len_p < nb_primes:
+        # Is n prime?
+        for i in p[:len_p]:
+            if n % i == 0:
+                break
+
+        # If no break occurred in the loop, we have a prime.
+        else:
+            p[len_p] = n
+            len_p += 1
+        n += 1
+
+    # Let's put the result in a python list:
```

Comment:
Also a good time to improve this (same for `primes.pyx`)
```suggestion
    # Let's convert the result to a Python list:
```

Reply:
I later see that the documentation says "The result is stored in the C array ``p`` during processing, and will be copied into a Python list at the end (line 26).", so a better suggestion would then be
```suggestion
    # Let's copy the result into a Python list:
```

Reply:
resolved [here](https://github.com/cython/cython/pull/4226/commits/64f9e0adb2604de2f789a24e47fc23ae74b0186b)